app/controllers/pet.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
should this file be called `get-single-post.php` or do you envisage other functions being in here